style.css

```diff
@@ -197,7 +204,6 @@ color: rgba(0, 0, 0, 0.46);
 }
 
 .some_text {
-  width: 400px;
   height: 220px;
   text-align: left;
 }
```

Comment:
.some_text ? Make it descriptive.
It's easier to do it if you set unique naming for eg. section.
- If the section is containing posts you can name it 'section-posts'
- If it has three posts each post can have a post-container
- If a post has a photo it's a post-img
- And a heading? post-heading
- Text? post-text
- etc...

Reply:
@sinkinshadow this is important ☝️

Reply:
roger that :)